odata-url-conventions/5 Query Options.md

```diff
@@ -1585,6 +1585,9 @@ Primitive literals can appear in the resource path as key property
 values, and in the query part, for example, as operands in
 [`$filter`](#SystemQueryOptionfilter) expressions. They are represented
 according to the `primitiveLiteral` rule in [OData-ABNF](#ODataABNF).
+The interpretation of a `timeOfDayLiteral` in which the `second` is omitted
+is not defined by this specification. For maximum interoperability, clients and servers
```

Comment:
"and servers", because servers can also produce resource path URLs in response payloads. Correct?